pkg/api/v20220904/openshiftinstallversions.go

```diff
@@ -0,0 +1,7 @@
+package v20220904
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// InstallOpenShiftVersions represents a List of OpenShift installable versions.
```

Comment:
Same in this commet: `InstallOpenShiftVersions` -> `InstallVersions` and also the filename has to change to `pkg/api/v20220904/installversions.go`.